.github/CONTRIBUTING.md

```diff
@@ -48,6 +48,10 @@ In addition to writing properly formatted commit messages, it's important to inc
 
 For package version upgrades and such a one-line commit message is usually sufficient.
 
+## Backporting changes
+
+To [backport a change into a release branch](https://nixos.org/nixpkgs/manual/#submitting-changes-stable-release-branches), take note of the commit in which the change was introduced into `master`. Then check out the release branch (e.g. `release-19.09`, _not_ the channel) and use `git cherry-pick -x <original commit>`. When opening your backport PR, make sure to select the release branch as the target branch of the PR, and link to the PR in which the original change was made to `master`.
```

Comment:
`_not_ the channel` is a bit too vague. If they don't know a release channel is named `release-19.09`, they probably won't know a channel is named `nixos-*` or `nixpkgs-*`. Add these two as examples of branches to not PR in to?

Reply:
Done.